oem/ibm/libpldmresponder/inband_code_update.cpp

```diff
@@ -387,6 +388,7 @@ void CodeUpdate::setVersions()
                                                     uint8_t(CodeUpdateState::
                                                                 START));
                                                 newImageId.clear();
+                                                imageActivationMatch.reset();
                                             }
                                         }
```

Comment:
why cant we clear the match after if & else if ? instead of at two places ?

Reply:
done